cmd/juju/application/deploy.go

```diff
@@ -464,12 +464,40 @@ media wiki' value. The same applies to single value options. For example,
 
 the value of 'my wiki' will be used.
 
-Use the '--resource' option to upload resources needed by the charm. This
-option may be repeated if multiple resources are needed:
+Use the '--resource' option to specify the resources you want to use for your charm.
+The format is
 
-  juju deploy foo --resource bar=/some/file.tgz --resource baz=./docs/cfg.xml
+    --resource <resource name>=<resource>
```

Comment:
would this be more helpful with the flag definition? but I can't find it for some reason.

Reply:
I found it, and I agree that would be a good place, but we don't do that for any other flag currently, so I think it's better to wait on that.